frodo.js

```diff
@@ -1,55 +1,50 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!